### DISCUSSION TITLE
如何合并dash下分离的音画文件？

### DISCUSSION THREAD

**Discussion by ID-izlq-Github** [the1812/Bilibili-Evolved#3789] [提问 / Q&A] [answered]
flv格式的4K及以上清晰度消失了，如果用剪辑软件合并会在HDR、杜比全景声、8K时遇到丢失相应格式、清晰度等问题

**Comment by Ethkuil** [comment, marked as answer]
强烈建议使用 [ffmpeg](https://ffmpeg.org/download.html)。许多视频处理软件底层都是用的这个。

然后这是很常见的需求，直接搜索就能搜到方法，比如[如何用ffmpeg合并音频和视频？ - ming-chatopera的回答 - 知乎](
https://www.zhihu.com/question/300182407/answer/722622224)。

**Reply by Ethkuil** [reply]
ffmpeg支持复制输入流，可以快速无损地完成合并。
